### PR TITLE
issue686: add AnnotationMemberDeclaration.removeDefaultValue

### LINKED ISSUE #686: Consider Adding AnnotationMemberDeclaration.removeDefaultValue
Similar to #681

### PR DESCRIPTION
**Pull request by ftomassetti**
fix #686

### CODE REVIEW

**Comment by ftomassetti** [comment]
rebased

File: javaparser-core/src/main/java/com/github/javaparser/ast/body/AnnotationMemberDeclaration.java [hunk 1]
```diff
@@ -142,6 +150,9 @@ public Type getType() {
      */
     public AnnotationMemberDeclaration setDefaultValue(Expression defaultValue) {
         notifyPropertyChange(ObservableProperty.DEFAULT_VALUE, this.defaultValue, defaultValue);
+        if (this.defaultValue != null) {
+            this.defaultValue.setParentNode(null);
+        }
```
Comment (matozoid) [inline javaparser-core/src/main/java/com/github/javaparser/ast/body/AnnotationMemberDeclaration.java:155]: I guess this is the code that we have to put in all setters?

**Comment by matozoid** [comment]
+1

File: javaparser-core/src/main/java/com/github/javaparser/ast/body/AnnotationMemberDeclaration.java (hunk 1 above)
Comment (ftomassetti) [reply javaparser-core/src/main/java/com/github/javaparser/ast/body/AnnotationMemberDeclaration.java:155]: yes, but maybe we can write an helper method in Node

**Pull request merged**